src/core/Akka/Actor/ActorCell.Children.cs

```diff
@@ -36,10 +36,10 @@ private IReadOnlyCollection<IActorRef> Children
         }
 
         private ImmutableDictionary<string, FunctionRef> FunctionRefs => Volatile.Read(ref _functionRefsDoNotCallMeDirectly);
-        internal bool TryGetFunctionRef(string name, out FunctionRef functionRef) =>
+        internal bool TryGetFunctionRef(string name, out FunctionRef? functionRef) =>
```

Comment:
All of these `TryGetValue<TKey, TValue>` methods have a signature of `bool `TryGetValue<TKey, TValue>(TKey key, out TValue? val)` in the .NET standard libraries - so I've brought the `TryGetFunctionRef` up to speed.